src/main/java/org/javarosa/core/model/ItemsetBinding.java

```diff
@@ -200,53 +201,60 @@ private SelectChoice getChoiceForTreeReference(FormDef formDef, DataInstance for
         return choice;
     }
 
-    // Build a map with keys for each value in the current answer. This will allow us to remove answers that are no
-    // longer available for selection because of an updated filter.
-    private Map<String, Boolean> initializeCurrentAnswerMap(FormDef formDef, TreeReference curQRef) {
-        Map<String, Boolean> currentAnswersInNewChoices = null;
+    /**
+     * Build a map with keys for each value in the current answer. This will allow us to remove answers that are no
+     * longer available for selection because of an updated filter and to bind selection(s) in the IAnswerData to Selection
+     * objects. The latter is necessary to get label text for the answers.
+     */
```

Comment:
I like how we’re slowly migrating these non-doc-comments to real, first class doc comments. I know you thought of me when you did this. :-)

Reply:
Yes, yes I did. 😊